hash/megadriv.xml

```diff
@@ -7624,10 +7624,21 @@ Unsupported [Menacer] peripheral
 		</part>
 	</software>
 
+	<software name="sonic1jp2" cloneof="sonic">
+		<!--	Mod of rev. A present in Sonic Mega Collection, which changes the spike damage behavior to be like that of later titles.	-->
+		<description>Sonic The Hedgehog (World, JP2, ripped from Sonic Mega Collection)</description>
+		<year>2002</year>
+		<publisher>Sega</publisher>
+		<part name="cart" interface="megadriv_cart">
+			<dataarea name="rom" width="16" endianness="big" size="524288">
+				<rom name="sonic the hedgehog (world) (Sonic mega collection).md" size="524288" crc="6382b2c5" sha1="95b6aac7e11bb2d908aafa06973ffeb45817a992"/>
+			</dataarea>
+		</part>
+	</software>
```

Comment:
I’m not sure this really belongs here, either.  We generally don’t add rips from emulated collections unless they’re placeholders for an undumped physical release.

Reply:
Maybe we should start doing it, since many classic game revisions have been released in collections for the last decades and won't be documented in MAME until it emulates console generations from the GameCube onwards.

New official alternate versions of games which people are actually playing on their real hardware.

Reply:
I think they're worth documenting.  They're not bad dumps, and this way MAME knows exactly what they are, and where they came from.  This might be less clear further down the line.